mapeditor/mainwindow.cpp

```diff
@@ -1181,3 +1181,13 @@ void MainWindow::on_actionPaste_triggered()
 	}
 }
 
+void MainWindow::keyPressEvent(QKeyEvent* event)
```

Comment:
according to our current [coding guidelines](https://wiki.vcmi.eu/Coding_guidelines#Where_to_put_spaces) should be:
```suggestion
void MainWindow::keyPressEvent(QKeyEvent * event)
```